docs/metrics.asciidoc

```diff
@@ -10,7 +10,7 @@ The .NET agent tracks certain system and application metrics.
 Some of them have built-in visualizations and some can only be visualized with custom Kibana dashboards.
 
 These metrics will be sent regularly to the APM Server and from there to Elasticsearch.
-You can adjust the interval with the setting <<config-metrics-interval>>.
+You can adjust the interval with the setting <<configuration.asciidoc#config-metrics-interval,`MetricsInterval`>>.
```

Comment:
https://github.com/elastic/apm-agent-dotnet/pull/369#issuecomment-509186961

Reply:
Yes, Greg is right, this link should just remain `<<config-metrics-interval>>`. Anchors and top level links are treated the same in the asciidoc source, so there's no need to specify "configuration.asciidoc#" .

Reply:
```suggestion
You can adjust the interval with the setting <<config-metrics-interval>>.
```

Reply:
The reason I thought it needs to be fixed is because Github doesn't render it correctly - `<<config-metrics-interval>>` at https://github.com/elastic/apm-agent-dotnet/blob/master/docs/metrics.asciidoc is rendered as 
```
https://github.com/elastic/apm-agent-dotnet/blob/master/docs/metrics.asciidoc#config-metrics-interval 
```
while it should be 
```
https://github.com/elastic/apm-agent-dotnet/blob/master/docs/configuration.asciidoc#config-metrics-interval
```
After the change Github does render it correctly (see https://github.com/SergeyKleyman/apm-agent-dotnet/blob/Fix_Metrics_docs/docs/metrics.asciidoc)

Is there a way to make it render correctly for both elastic.co and Github?

Reply:
Unfortunately no. We try to push people to read documentation on the website, and not on GitHub (hence the "NOTE" at the top of the page). While some internal doc links may work in both, all external doc links, like the link to the metricbeat reference in your link above, will only work on our website as well.